.drone.jsonnet

```diff
@@ -403,6 +403,16 @@ local integration_qemu_encrypted_vip = Step("e2e-encrypted-vip", target="e2e-qem
         "IMAGE_REGISTRY": local_registry,
 });
 
+local integration_qemu_csi = Step("e2e-csi", target="e2e-qemu", privileged=true, depends_on=[load_artifacts], environment={
+        "IMAGE_REGISTRY": local_registry,
+        "QEMU_WORKERS": "3",
+        "QEMU_CPUS_WORKERS": "4",
+        "QEMU_MEMORY_WORKERS": "5120",
+        "QEMU_EXTRA_DISKS": "1",
+        "QEMU_EXTRA_DISKS_SIZE": "12288",
+        "WITH_TEST": "run_csi_tests",
```

Comment:
do you want to run short integration test here? as it runs anyways before the Ceph is deployed, it doesn't give much new information?

Reply:
fixed